app/index.html

```diff
@@ -43,6 +35,10 @@
     <!-- Main app module -->
     <script src="app.js"></script>
 
+    <!-- Header module -->
```

Comment:
A minor point, but you may want to change the comment to `<!--Layout module-->`
It's just a header right now, but later you may want to add more layout-type elements like a footer or navbar, and that would keep the heading more generalized